astropy/units/tests/test_units.py

```diff
@@ -160,8 +160,6 @@ def test_unknown_unit():
 
 
 def test_multiple_solidus():
-    assert u.Unit("m/s/kg").to_string() == u.m / u.s / u.kg
```

Comment:
Is this removal on purpose? It sounds like the multiple calls were needed for a test named `test_multiple_solidus` but I am just guessing.

Reply:
It just duplicates the next test, but without a `catch_warnings`